src/Controls/src/Core/Handlers/Items/Android/SelectableViewHolder.cs

```diff
@@ -64,7 +64,7 @@ void SetSelectionStates(bool isSelected)
 			{
 				return;
 			}
-			if (PlatformVersion.IsAtLeast(Android.OS.BuildVersionCodes.M))
+			if (OperatingSystem.IsAndroidVersionAtLeast(23))
```

Comment:
This is up to you if you think it looks any better:
```csharp
if (OperatingSystem.IsAndroidVersionAtLeast((int)Android.OS.BuildVersionCodes.M))
```
I checked the IL, and the C# compiler puts a 23 `const` so there is no performance impact doing this.

Reply:
I need to check again, but I think that the analyzer didn't like it as a const field. But maybe I didn't reload enough times...

Reply:
Oh yeah, it's possible this would break the analyzer...